docs/src/hal/tools.adoc

```diff
@@ -368,9 +368,9 @@ function report the function as "Unknown".
 
 [NOTE]
 `halreport` generates a connections report for a running HAL
-application to aid in designing and verifying connections.  Pin
-types and current values are not shown.  For this information use
+application to aid in designing and verifying connections.
+Pin types and current values are not shown.  For this information use
 applications like `halshow`, `halmeter`, `halscope` or the 'show'
 command available with command-line `halcmd` program.
```

Comment:
```suggestion
`halreport` generates a connections,  pin types, and current values
report for a running HAL application to aid in designing and verifying
connections. +
Use this information with applications like `halshow`, `halmeter`,
`halscope` or the  `halcmd  show` command in a terminal.
```

Reply:
I am not fully happy about this paragraph (also not with the original). One setence the change by @silopolis is left out - that is fine? I cannot judge myself, really. Also, can we have the report mentioned earlier? There is otherwise the danger of this sentence being missinterpreted.

Reply:
Why aren't you happy with the original one?

Reply:
> I am not fully happy about this paragraph (also not with the original). One setence the change by @silopolis is left out - that is fine? I cannot judge myself, really. Also, can we have the report mentioned earlier? There is otherwise the danger of this sentence being missinterpreted.

Realised I made a mistake in my first proposal. Edited and corrected.
About the original "For this information..." sentence, it just doesn't seem english or make sense to me, but I'm french 😅

Reply:
Adopted @silopolis' version but removed the "Note:" and added a sentence.